Source/WebKit/UIProcess/RemoteLayerTree/mac/RemoteScrollingCoordinatorProxyMac.mm

```diff
@@ -280,6 +280,16 @@
     scrollingTree()->unlockLayersForHitTesting();
 }
 
+void RemoteScrollingCoordinatorProxyMac::animationEffectStackWasAddedToLayer(PlatformLayer*, Ref<RemoteAcceleratedEffectStack> effects)
+{
+    m_wheelEventDispatcher->animationEffectStackWasAddedToLayer(WTFMove(effects));
```

Comment:
`m_wheelEventDispatcher` is a confusing name here. Who'd have thought that a mouse wheel thing cared about animations.